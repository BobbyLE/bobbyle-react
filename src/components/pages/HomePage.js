import React from 'react';

const HomePage = () => {
  return (
    <div className="content-container">Homepage</div>
  )
};

export default HomePage;