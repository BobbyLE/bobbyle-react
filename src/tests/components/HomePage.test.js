import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../components/HomePage';

test('should correctly render HomePage', () => {
  const wrapper = shallow(<HomePage />);
  expect(wrapper).toMatchSnapshot();
});
