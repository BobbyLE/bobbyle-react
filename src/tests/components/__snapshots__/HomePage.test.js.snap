// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render HomePage 1`] = `
<div>
  Homepage
</div>
`;
